MuonAnalysis/MuonAssociators/src/PropagateToMuonSetup.cc

```diff
@@ -0,0 +1,78 @@
+#include "MuonAnalysis/MuonAssociators/interface/PropagateToMuonSetup.h"
+
+#include <cmath>
+
+#include "DataFormats/RecoCandidate/interface/RecoCandidate.h"
+
+#include "DataFormats/GeometrySurface/interface/TrapezoidalPlaneBounds.h"
+#include "MagneticField/Records/interface/IdealMagneticFieldRecord.h"
+#include "RecoMuon/Records/interface/MuonRecoGeometryRecord.h"
+#include "TrackingTools/DetLayers/interface/DetLayer.h"
+#include "TrackingTools/KalmanUpdators/interface/Chi2MeasurementEstimator.h"
+#include "TrackingTools/Records/interface/TrackingComponentsRecord.h"
+#include "TrackingTools/TrajectoryState/interface/TrajectoryStateTransform.h"
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
```

Comment:
This doesn't look to be used here, rather "FWCore/Utilities/interface/Exception.h"

Reply:
This was indeed superfluous, removed.